.eslintrc.js

```diff
@@ -1313,7 +1313,7 @@ module.exports = {
             'x-pack/plugins/security_solution_serverless/**/*.{js,mjs,ts,tsx}',
             'x-pack/plugins/cases/**/*.{js,mjs,ts,tsx}',
             'x-pack/plugins/ecs_data_quality_dashboard/**/*.{js,mjs,ts,tsx}',
-            'x-pack/packages/kbn-data-stream-adapter/**/*.{js,mjs,ts,tsx}',
+            'x-pack/x-pack/solutions/security/packages/kbn-data-stream-adapter/**/*.{js,mjs,ts,tsx}',
```

Comment:
Why is there a nested x-pack?

Reply:
Error in my relocation script, will re-run to auto fix. Sorry about that